server/server.go

```diff
@@ -0,0 +1,213 @@
+package server
+
+import (
+	"errors"
+	"expvar"
+	"net"
+	"net/http"
+	"net/http/pprof"
+	"os"
+	"path/filepath"
+
+	"github.com/boltdb/bolt"
+	containers "github.com/containerd/containerd/api/services/containers/v1"
```

Comment:
this is more of a nit and `containersapi` is verbose.  i'm not sure it matters and only came into play when you import the `containers` and api packages